package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "xcomponentjsapi",
+  "version": "1.0.0",
+  "description": "js api",
+  "main": "index.html",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/xcomponent/ReactiveXComponent.js.git"
+  },
+  "keywords": [
+    "rx",
+    "api",
+    "js",
+    "require",
+    "requirejs",
+    "karma"
+  ],
+  "author": "hazem mbarek",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/xcomponent/ReactiveXComponent.js/issues"
+  },
+  "homepage": "https://github.com/xcomponent/ReactiveXComponent.js#readme",
+  "dependencies": {
+    "requirejs": "^2.2.0"
+  },
+  "devDependencies": {
+    "jasmine-core": "^2.4.1",
+    "karma": "^0.13.22",
+    "karma-chrome-launcher": "^1.0.1",
+    "karma-coverage": "^1.0.0",
+    "karma-jasmine": "^1.0.2",
+    "karma-requirejs": "^1.0.0"
```

Comment:
add this dependency:
"karma-junit-reporter": "*"
Used to publish junit report in jenkins